neon_diana_utils/configuration.py

```diff
@@ -337,7 +337,7 @@ def update_rmq_config(config_file: str = None) -> str:
     with open(config_file) as f:
         real_config = json.load(f)
     new_config = generate_rmq_config("", "")
-    existing_users = (user['name'] for user in real_config['users'])
+    existing_users = [user['name'] for user in real_config['users']]
```

Comment:
I suppose it should make no difference here in practice, but that would remove any duplicates from an existing config (i.e. one that has been manually modified) so that's a good idea

Reply:
On second read, it would make no difference since we only append new users here.